lib/postRunHook.js

```diff
@@ -0,0 +1,16 @@
+var Q = require('q');
+var events = require('./utils/events');
+var os = require('os');
+
+module.exports = function(context) {
+
+    var pluginName = require('../package.json').name;
+
+    // Let the user know how he can kill the LiveReload/BrowserSync server
+    var msg = os.EOL;
+    msg += pluginName + ' - ' + 'LiveReload server running: ' + os.EOL + 'To kill it, please use: "CTRL+C" ' + os.EOL;
+
+    console.log(msg);
```

Comment:
Ideally, you can have a utility `log` function that appends plugin name instead of all the code doing this.